library/advanced-search.en.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: Advanced Search
+Category: Library
+Template: document
+Tags: support
+Date: 2017-07-20
+Slug: advanced-search
+Summary:
```

Comment:
Summary: Some details about features of the search engine built into Opendata.swiss

?